source/wp-content/themes/wporg-developer-blog/parts/post-meta.html

```diff
@@ -0,0 +1,27 @@
+<!-- wp:group {"style":{"spacing":{"blockGap":"var:preset|spacing|10"}},"className":"remove-row-gap","layout":{"type":"flex","flexWrap":"wrap"}} -->
+<div class="wp-block-group remove-row-gap">
+    
+    <!-- wp:group {"style":{"spacing":{"blockGap":"0.25rem"}},"layout":{"type":"flex","flexWrap":"nowrap"},"fontSize":"small"} -->
+    <div class="wp-block-group has-small-font-size">
+        <!-- wp:paragraph -->
+        <p>By</p>
+        <!-- /wp:paragraph -->
+
+        <!-- wp:post-author-name {"isLink":true} /-->
+    </div>
+    <!-- /wp:group -->
+    
+    <!-- wp:paragraph -->
+    <p>·</p>
```

Comment:
This being a `p` tag feels wrong. Consider a different HTML structure.

Reply:
Yeah I agree, but this is really the only way to do it with blocks. 

That said, I decided to swap this out with a CSS approach similar to how the new pagination works. [9c3cc74](https://github.com/WordPress/wporg-developer-blog/pull/47/commits/9c3cc740c34312be1c4592fd9170b08e0518e996) addresses this.